worlds/witness/test/test_early_good_item.py

```diff
@@ -0,0 +1,109 @@
+from BaseClasses import ItemClassification, LocationProgressType
+from Fill import distribute_items_restrictive
+
+from ..test import WitnessTestBase
+
+
+class TestEarlySymbolItemFalse(WitnessTestBase):
+    options = {
+        "early_good_items": {},
+
+        "shuffle_symbols": True,
+        "shuffle_doors": "off",
+        "shuffle_boat": False,
+        "shuffle_lasers": False,
+        "obelisk_keys": False,
+    }
+
+    def setUp(self) -> None:
+        if self.auto_construct:
+            self.world_setup(seed=1)  # Magic seed to prevent false positive
+
+    def test_early_good_item(self) -> None:
+        distribute_items_restrictive(self.multiworld)
+
+        gate_open = self.multiworld.get_location("Tutorial Gate Open", 1)
+
+        self.assertFalse(
```

Comment:
So this isn't technically testing the right thing, but I guess that's what the magic seed is for, which seems volatile but 🤷

Reply:
Yeah, there's not really a great way to test "This thing that should only have a chance to happen, doesn't always happen" without doing 100 gens or something unless you do a magic seed